spec.html

```diff
@@ -27209,31 +27215,257 @@
+                <td>
+                  `"Jun"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  6
+                </td>
+                <td>
+                  `"Jul"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  7
+                </td>
+                <td>
+                  `"Aug"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  8
+                </td>
+                <td>
+                  `"Sep"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  9
+                </td>
+                <td>
+                  `"Oct"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  10
+                </td>
+                <td>
+                  `"Nov"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  11
+                </td>
+                <td>
+                  `"Dec"`
+                </td>
+              </tr>
+              </tbody>
+            </table>
+          </emu-table>
+        </emu-clause>
+
+        <emu-clause id="sec-timezoneestring" aoid="TimeZoneString">
+          <h1>Runtime Semantics: TimeZoneString( _tv_, )</h1>
+          <p>The following steps are performed:</p>
+          <emu-alg>
+            1. Assert: Type(_tv_) is Number.
+            1. Assert: _tv_ is not *NaN*.
+            1. Let _offset_ be (LocalTZA + DaylightSavingTA(_tv_)) / msPerSecond.
+            1. Assert: _offset_ is an integer, and -100000 &lt; _offset_ &lt; 100000.
+            1. If _offset_ &ge; 0, let _offsetSign_ be `"+"`; otherwise, let _offsetSign_ be `"-"`.
+            1. Let _offsetString_ be abs(_offset_) formatted as a four-digit number, padded to the left with zeros if necessary.
+            1. Let _tzName_ be an implementation-defined string, either `""` or a string of the form `" ("` _name_ `")"` where _name_ is an implementation-defined timezone name.
+            1. Return the String value formed by concatenating _offsetSign_, _offsetString_, and _tzName_.
+          </emu-alg>
+        </emu-clause>
+
         <!-- es6num="20.3.4.41.1" -->
         <emu-clause id="sec-todatestring" aoid="ToDateString">
           <h1>Runtime Semantics: ToDateString( _tv_ )</h1>
           <p>The following steps are performed:</p>
           <emu-alg>
             1. Assert: Type(_tv_) is Number.
             1. If _tv_ is *NaN*, return `"Invalid Date"`.
-            1. Return an implementation-dependent String value that represents _tv_ as a date and time in the current time zone using a convenient, human-readable form.
+            1. Let _t_ be LocalTime(_tv_).
+            1. Return the String value formed by concatenating DateString(_t_), `" "`, TimeString(_t_), and TimeZoneString(_tv_).
```

Comment:
for the space, the convention i've seen (and followed in padStart/padEnd) to avoid whitespace ambiguity is `" " (a String consisting of 0x0020 SPACE)`